aws-ts-netlify-cms-and-oauth/cms/public/config.yml

```diff
@@ -0,0 +1,95 @@
+backend:
+  name: github
+  # Replace this with the Github repo you want to make change
+  repo: github-username/target-github-repo
+  # This site_domain and base_url are sudo domains
+  # Replace site_domain with targetDomain pulumi stack configuration inside cms/infrastructure folder
+  # Replace base_url with targetDomain pulumi stack configuration inside cms-oauth/infrastructure folder
+  site_domain: https://some-cms-domain.pulumi-demos.com  
+  base_url: https://some-oauth-domain.pulumi-demos.com 
+  
+publish_mode: editorial_workflow
+
+# the media_folder's path is relative to this 
+media_folder: "../../../static/images"
+public_folder: "/images"
+
+collections: # A list of collection the CMS should be able to edit
```

Comment:
Isn't this setup specific to our use? Let's change this to be a very basic example, probably with just one entry under it or even nothing, and adding a comment to point to the Netlify CMS' own docs website.

Reply:
Deleted the collection and set place-holder words for media and public folder